packages/cli/generation/ir-migrations/src/migrations/v50-to-v49/migrateFromV50ToV49.ts

```diff
@@ -33,7 +37,9 @@ export const V50_TO_V49_MIGRATION: IrMigration<
         [GeneratorName.RUBY_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
         [GeneratorName.RUBY_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
         [GeneratorName.CSHARP_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
-        [GeneratorName.CSHARP_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR
+        [GeneratorName.CSHARP_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.SWIFT_MODEL]: GeneratorWasNotCreatedYet,
```

Comment:
Let's tag `0.0.0` for these since this is the latest IR (the first one the generator will consume). You can see this in the other migration files when the generator was first introduced (e.g. IRv32 for Ruby).

Reply:
cool only doing for model since there might be another version of the IR before we build that